playground/mocks/data/idp/idx/identify-unknown-user.json

```diff
@@ -52,7 +52,7 @@
     "type": "array",
     "value": [
       {
-        "message": "There is no account with the email <strong> test@rain.com </strong>. <a href=\"#\" class=\"js-sign-up\"> Sign up </a> for an account",
+        "message": "There is no account with the email test@rain.com. Error: idx.unknown.user",
```

Comment:
can you use the exact error, that server is sending ?

Reply:
I hope server doesn't send `Error: idx.unknown.user` with the message..

Reply:
yeah, I am not sure why we need `Error: idx.unknown.user` as part of the message.